src/framework/mod.rs

```diff
@@ -644,6 +767,23 @@ impl Framework {
 
     /// Specify the function to be called after every command's execution.
     /// Fourth argument exists if command returned an error which you can handle.
+    ///
+    /// # Examples
+    ///
+    /// Using after to log command usage:
+    ///
+    /// ```rust
+    /// # use serenity::Client;
+    /// # let mut client = Client::login("token");
+    ///
```

Comment:
As with above, if I'm not mistaken this empty line will cause extra padding at the top of the codeblock -- I think it'd look cleaner if this were `/// #`, instead of removing the line altogether.